src/component/Navbar.js

```diff
@@ -22,12 +22,14 @@ import { FaBars } from "react-icons/fa";
 const NavBar = (props) => {
   const [nav, setNav] = useState(false);
   const fAuth = getAuth();
+  const navigate = useNavigate();
   const fireBaseSignOut = async () => {
     signOut(fAuth)
       .then(() => {
         console.log("success");
+        props.setUser(null);
         navigate("/login");
-        props.setLoggedIn(false);
+        // props.setLoggedIn(false);
```

Comment:
why is this commented out?

Reply:
My bad. I didn't see it was commented out.